examples/whisper.py

```diff
@@ -32,7 +41,7 @@ def __call__(self, x:Tensor, xa:Optional[Tensor]=None, mask:Optional[Tensor]=Non
     return self.out(wv)
 
   def qkv_attention(self, q, k, v, mask=None):
-    n_batch, n_ctx, n_state = q.shape
+    _, n_ctx, n_state = q.shape
```

Comment:
Instead of ```qkv_attention``` you may call the newly merged ```Tensor.scaled_dot_product_attention```. Like here [7fe8ce0](https://github.com/tinygrad/tinygrad/pull/1371/commits/7fe8ce05a7f508b9f623ccec46146051f4781a0a)

Reply:
Done!